client/src/featureform/register.py

```diff
@@ -83,6 +83,14 @@
 NON_INFERENCE_STORES = [s3_config.type()]
 
 
+def set_tags_properties(tags: List[str], properties: dict):
```

Comment:
super minor but I feel like tags & properties should never be none, just an empty list